### PR TITLE
All change options 

### LINKED ISSUE #63: all properties of events can now be changed
as of release [v3.1.0](https://github.com/cpvalente/ontime/releases/tag/v3.1.0) all properties of an event can now be changed
this should be added to the module

 should be a riminder about the throttling on the sensitive endpoints ?

### PR DESCRIPTION
**Pull request by alex-Arc**
closes #63

### CODE REVIEW

**Comment by alex-Arc** [comment]
@cpvalente this is ready

File: src/v3/actions/changePicker.ts [hunk 1]
```diff
@@ -72,6 +62,82 @@ export function changePicker(
...
+			step: 1,
+			isVisible: (opts) => Array.isArray(opts.properties) && opts.properties.includes('timeStart'),
+		},
+		{
+			type: 'number',
+			label: 'End Time',
+			id: 'timeEnd',
+			default: 0,
+			min: 0,
+			max: 86399,
```
Comment (cpvalente) [inline src/v3/actions/changePicker.ts:91]: Looks like this should be a constant somewhere so we can label it?

Also, what is the unit here? Is it clear to the user?

**Review by cpvalente — APPROVED** [review]

**Comment by cpvalente** [comment]
@alex-Arc , is this ready to merge?

**Comment by alex-Arc** [comment]
yes, thank you

**Pull request merged**